README.md

```diff
@@ -68,6 +67,10 @@ $ terraform init
 $ terraform apply
 ```
 
+##### Important (after initial `terraform apply`)
+
+The generated base `.tfstate` is not stored in the remote state S3 bucket. Ensure the base `.tfstate` is checked into your infrastructure repo. The default Terraform `.gitignore` [generated by GitHub](https://github.com/github/gitignore/blob/master/Terraform.gitignore) will ignore all `.tfstate` files; you'll need to modify this!
+
```

Comment:
Maybe add a note that in the worst case, if the state does become lost, you can probably reconstruct it fairly accurately with `terraform import` for the relatively simple case of the `base` directory.

Reply:
And maybe an additional warning to verify that other state files may contain secrets (which should not be committed). 

But otherwise, I think its fine.